specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json

```diff
@@ -507,6 +507,53 @@
         }
       }
     },
+    "AzureStorageInfoValue": {
+      "description": "Azure Files or Blob Storage access information value for dictionary storage.",
+      "type": "object",
+      "properties": {
```

Comment:
nit: Please check if any of these properties are readOnly

Reply:
State should be readOnly, fixed in latest commit.